src/main/kotlin/stringcalculator/StringCalculator.kt

```diff
@@ -0,0 +1,29 @@
+package stringcalculator
+
+
+class StringCalculator {
+    companion object {
+        fun calculate(input: String?): Int {
+            require(!input.isNullOrBlank())
+
```

Comment:
어떤 이유에서 예외가 발생했는 지 Exception 메시지를 적어주면 디버깅하기 용이해집니다 :)
아래는 예시이니, 참고해서 작성해주세요 :)
```suggestion
            require(!input.isNullOrBlank()) { "입력 값이 null이거나 EmptyString이면 안됩니다. 입력 값 : $input" }

```